RecoTracker/LST/interface/LSTOutput.h

```diff
@@ -0,0 +1,38 @@
+#ifndef RecoTracker_LST_interface_LSTOutput_h
+#define RecoTracker_LST_interface_LSTOutput_h
+
+#include <memory>
+#include <vector>
+
+class LSTOutput {
+public:
+  LSTOutput() = default;
+  LSTOutput(std::vector<std::vector<unsigned int>> const hitIdx,
+            std::vector<unsigned int> const len,
+            std::vector<int> const seedIdx,
+            std::vector<short> const trackCandidateType)
+      : hitIdx_(std::move(hitIdx)),
+        len_(std::move(len)),
+        seedIdx_(std::move(seedIdx)),
+        trackCandidateType_(std::move(trackCandidateType)) {}
+
+  enum LSTTCType { T5 = 4, pT3 = 5, pT5 = 7, pLS = 8 };
```

Comment:
@VourMa 
this is now the same as `LSTObjType`.
Is there a good reason to duplicate?

RecoTracker/LSTCore/interface/Common.h seems more authoritative than this location

Reply:
I initially thought that the LSTObjType would include more types (not only those included in TCs) but it turns out that it doesn't. However, `LSTOutput.h` is a CMSSW-only module, and I am not sure we want it to have interface with `LSTCore`-only header files -the only way they interface till now is through the `LST` class If it's ok to "export" an additional dependency from the `LSTCore` package, then we could remove this indeed.

Reply:
I'm mainly concerned about expanding standalone being dependent on more of CMSSW packages.

I'm not sure about cases where dependency of CMSSW-specific LST or even remotely LST package on LSTCore/interface can be a problem.
Do you have some hypothetical examples?

Reply:
No, I don't have any specific example. I can give a try to switching to the enum in `Common.h` while resolving conflicts.